src/components/LoadingSkeletons/BarChartLoadingSkeleton.tsx

```diff
@@ -0,0 +1,96 @@
+import React, { FunctionComponent } from 'react';
+import LoadingMessage from '../LoadingMessage/LoadingMessage';
+import { backgroundGray } from './LoadingSkeletonsSvgUtil';
+import LoadingSkeleton, {
+	ILoadingSkeletonProps,
+	IStandardSkeleton,
+} from './LoadingSkeleton';
+
+export const BarChartSkeleton: FunctionComponent<IStandardSkeleton> = props => {
+	const { width, height, className } = { ...props };
+
+	return (
+		<div
+			data-test-id='loadingSkeleton-BarChartSkeleton'
+			className={backgroundGray('&', className)}
+		>
+			<svg
+				data-test-id='loadingSkeleton-BarChartSkeleton-svg'
+				width={width}
+				height={height}
+				version='1.1'
+			>
+				<g
+					id='Details'
+					stroke='none'
+					strokeWidth='1'
+					fill='none'
+					fillRule='evenodd'
+				>
+					<g
+						id='Analytics-Tab_Skeleton-DM'
+						transform='translate(-93.000000, -723.000000)'
+					>
+						<g id='Bar-Chart' transform='translate(93.000000, 723.000000)'>
+							<path
+								className={backgroundGray('&', className)}
```

Comment:
class name should be something like this:
`
const cx = lucidClassNames.bind('&-BarChartSkeleton');

<path
								className={cx('&', className)}

`

Reply:
Done